src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,67 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
 
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
-        System.out.println(movieService.get(fastAndFurious.getId()));
-        movieService.getAll().forEach(System.out::println);
+
+        //cascades
+        Movie fastAndStupid = new Movie("Fast and Stupid");
+        fastAndStupid.setDescription("An action comedy movie about romantic relationships");
+        Movie javaMovie = new Movie("MyJavaJourney");
+        javaMovie.setDescription("A thriller about setting out on a career in java");
```

Comment:
This block of code contains movie creation that is not being used or persisted. Ensure that you only create and persist entities that are required by the task. If these movies are meant to be used, they should be added to the database with `movieService.add()`.